### PR TITLE
Prepared captcha, dynamic price plan

### PR DESCRIPTION
**Pull request by tenisakb**

### CODE REVIEW

File: src/components/signup/record-recovery-phrase-slide.tsx [hunk 1]
```diff
@@ -78,6 +79,8 @@ interface RecordRecoveryPhraseState {
   isTermsChecked;
 }
 
+let recaptcha;
```
Comment (EdmundMai) [inline src/components/signup/record-recovery-phrase-slide.tsx]: don't do this. use `useRef`

Comment (EdmundMai) [reply src/components/signup/record-recovery-phrase-slide.tsx]: convert this component to use the new React syntax (useState, useRef, useEffect, etc) if you need to

File: src/config/index.ts [hunk 2]
```diff
@@ -196,3 +198,69 @@ export const theme = {
   lineHeight: "normal",
   letterSpacing: "normal"
 };
+
+export const subscriptionList = [
```
Comment (EdmundMai) [inline src/config/index.ts]: SUBSCRIPTION_LIST

File: src/components/shared/subscription.tsx [hunk 3]
```diff
@@ -2,7 +2,12 @@ import _ from "lodash";
 import React from "react";
 import styled, { ThemeProvider } from "styled-components";
 
-import { SUBSCRIPTION_DESKTOP_WIDTH, MOBILE_WIDTH, theme } from "../../config";
+import {
+  SUBSCRIPTION_DESKTOP_WIDTH,
+  subscriptionList,
```
Comment (EdmundMai) [inline src/components/shared/subscription.tsx]: SUBSCRIPTION_LIST

File: src/components/signup/index.tsx [hunk 4]
```diff
@@ -32,6 +34,7 @@ const SignUp = ({
     openMetamask={openMetamask}
     pollPayment={pollPayment}
     goBack={goBack}
+    subscription={subscriptionList[0]}
```
Comment (EdmundMai) [inline src/components/signup/index.tsx]: when do you plan on making this dynamic?

File: src/components/shared/subscription.tsx (hunk 3 above)
Comment (tenisakb) [reply src/components/shared/subscription.tsx]: fixed

File: src/components/signup/index.tsx (hunk 4 above)
Comment (tenisakb) [reply src/components/signup/index.tsx]: soon :)) today

File: src/components/signup/record-recovery-phrase-slide.tsx (hunk 1 above)
Comment (tenisakb) [reply src/components/signup/record-recovery-phrase-slide.tsx]: fixed

File: src/components/signup/record-recovery-phrase-slide.tsx [hunk 5]
```diff
@@ -97,74 +98,62 @@ class RecordRecoveryPhraseSlide extends Component<
...
+            <Box key={i}>
+              {i + 1}. {word}
+            </Box>
+          ))}
+        </Grid>
+        <DownloadButton onClick={() => downloadCsv(mnemonic)}>
+          Download phrase as CSV
+        </DownloadButton>
+        <Recaptcha
+          ref={ref => (recaptcha = ref)}
```
Comment (EdmundMai) [inline src/components/signup/record-recovery-phrase-slide.tsx]: you can just do `ref={recaptcha}`

File: src/components/signup/record-recovery-phrase-slide.tsx [hunk 6]
```diff
@@ -69,24 +70,24 @@ const Box = styled.div`
...
-  next;
-  mnemonic;
-}
-
-interface RecordRecoveryPhraseState {
-  isTermsChecked;
-}
+const RecordRecoveryPhraseSlide = ({ next, mnemonic }) => {
+  const [isTermsChecked, setIsTermsChecked] = useState();
+  let recaptcha = useRef<any>(null);
```
Comment (EdmundMai) [inline src/components/signup/record-recovery-phrase-slide.tsx]: const recaptcha

**Comment by EdmundMai** [comment]
Please fix conflicts

**Pull request merged**